db-test/src/androidTest/java/com/islamversity/db_test/db_filler/InsertQuranTextTest.kt

```diff
@@ -36,15 +30,16 @@ class InsertQuranTextTest {
             dbName,
             useNoBackupDirectory = true
         )
-        db = Main(driver)
+
+        db = createMainDB(driver)
     }
 
     @After
     fun tearDown() {
-        val file = File(InstrumentationRegistry.getInstrumentation().targetContext.getNoBackupFilesDir(), dbName)
-        assertThat(file.exists()).isTrue()
-        driver.close()
-        file.delete()
+//        val file = File(InstrumentationRegistry.getInstrumentation().targetContext.getNoBackupFilesDir(), dbName)
+//        assertThat(file.exists()).isTrue()
+//        driver.close()
+//        file.delete()
     }
 
     @Test
```

Comment:
There is no assertion

Reply:
This is not actually a test!
this is the fastest way I could create an Android compatible database (to ship with the app).
after this function has run
I'll go the database location on the emulator and copy it to our assets! 🤦